lib/compat/wordpress-6.3/rest-api.php

```diff
@@ -76,3 +76,34 @@ function gutenberg_update_global_styles_rest_controller( $args, $post_type ) {
 	return $args;
 }
 add_filter( 'register_post_type_args', 'gutenberg_update_global_styles_rest_controller', 10, 2 );
+
+/**
+ * Add the `modified` value to the `wp_template` schema.
+ *
+ * @since 6.3.0 Added 'modified' property and response value.
+ */
+function add_modified_wp_template_schema() {
+		register_rest_field(
+			array( 'wp_template', 'wp_template_part' ),
+			'modified',
+			array(
+				'schema'       => array(
+					'description' => __( "The date the post was last modified, in the site's timezone.", 'gutenberg' ),
```

Comment:
Tiny nit: should this be `template` instead of `post`?

```suggestion
					'description' => __( "The date the template was last modified, in the site's timezone.", 'gutenberg' ),
```